stress_test_model_cb.R

```diff
@@ -470,16 +487,16 @@ for (i in seq(1, nrow(transition_scenarios))) {
     # NOTE: this assumes emissions factors stay constant after forecast and prod not continued
     tidyr::fill(
       company_id, pd, net_profit_margin, debt_equity_ratio, volatility,
-      ald_emissions_factor, ald_emissions_factor_unit, ald_production_unit,
+      #ald_emissions_factor, ald_emissions_factor_unit, ald_production_unit,
```

Comment:
please no commenting out. Again I noticed that handling for equity_annual_profits was not adjusted and wonder if that is intended